ckan/cli/cli.py

```diff
@@ -0,0 +1,32 @@
+# encoding: utf-8
+
+import os
```

Comment:
There's quite a lot of unused imports in the different files, can you remove them? Your IDE/editor should be able to help with that.